vmdk_plugin/utils/test_util/VolumeCreationTestUtil.go

```diff
@@ -49,3 +52,83 @@ func DeleteVolume(name string, ip string) ([]byte, error) {
 func InvokeCommand(remoteHostIP string, cmd string) ([]byte, error) {
 	return exec.Command("/usr/bin/ssh", append(SSH_OPTS, "root@"+remoteHostIP, cmd)...).CombinedOutput()
 }
+
+// returns disk format of volume using Admin cli
+func GetDiskFormatUsingAdminCli(volName string, hostname string) string {
```

Comment:
I am afraid this approach will slow down the test significantly. Can we get ALL needed info from Admin CLI in one shot instead of multiple invocations of Admin CLI ?